pkgs/tools/filesystems/irods/default.nix

```diff
@@ -35,18 +35,18 @@ rec {
 
     postPatch = common.postPatch + ''
       patchShebangs ./test
-      substituteInPlace plugins/database/CMakeLists.txt --replace "COMMAND cpp" "COMMAND ${gcc.cc}/bin/cpp"
+      substituteInPlace plugins/database/CMakeLists.txt --replace-warn "COMMAND cpp" "COMMAND ${gcc.cc}/bin/cpp"
       for file in unit_tests/cmake/test_config/*.cmake
       do
-        substituteInPlace $file --replace "CATCH2}/include" "CATCH2}/include/catch2"
+        substituteInPlace $file --replace-quiet "CATCH2}/include" "CATCH2}/include/catch2"
       done
       export cmakeFlags="$cmakeFlags
         -DCMAKE_EXE_LINKER_FLAGS=-Wl,-rpath,$out/lib
         -DCMAKE_MODULE_LINKER_FLAGS=-Wl,-rpath,$out/lib
         -DCMAKE_SHARED_LINKER_FLAGS=-Wl,-rpath,$out/lib
         "
 
-      substituteInPlace server/auth/CMakeLists.txt --replace SETUID ""
+      substituteInPlace server/auth/CMakeLists.txt --replace-warn SETUID ""
     '';
```

Comment:
why `warn` and `quiet` instead of `fail`?

Reply:
- replaced the `warn` by `fail`
- let the `quiet` in place as this makes the package more robust to further updates (the replace is inside a loop and normally fails for most of the files)